cowdao_cowpy/contracts/sign.py

```diff
@@ -43,24 +43,36 @@ class EcdsaSignature:
     scheme: SigningScheme
     data: str
 
+    def to_string(self) -> str:
+        return self.data
+
 
 @dataclass
 class Eip1271SignatureData:
     verifier: str
     signature: bytes
 
+    def to_string(self) -> str:
+        return str(self.signature.hex())
```

Comment:
[bytes.hex](https://docs.python.org/3/library/stdtypes.html#bytes.hex) already returns `str`

Reply:
Has to be explicitly called otherwise cannot do starts and ends with no? 
It's a Hex  type irrc no?